build_tools/cmake/build_all.sh

```diff
@@ -54,6 +54,10 @@ declare -a CMAKE_ARGS=(
   "-DIREE_HAL_DRIVER_CUDA=ON"
   "-DIREE_TARGET_BACKEND_CUDA=ON"
 
+  # Enable WebGPU compiler builds and tests. All deps get fetched as needed,
+  # but some of the deps are too large to enable by default for all developers.
+  "-DIREE_TARGET_BACKEND_WEBGPU=ON"
```

Comment:
Do we want to also enable this in `build_and_test_asan.sh` and `build_and_test_tsan.sh`?

Reply:
Oo I think that's a good idea. I'll try it.

Reply:
Hmm... the ASan build (not even getting to tests) fails with

```
_deps/tint-build/src/tint/libtint_diagnostic_utils.a(source.cc.o): In function `tint::(anonymous namespace)::ParseLineBreak(std::basic_string_view<char, std::char_traits<char> >, unsigned long, bool*, unsigned long*)':
/work/build-asan/_deps/tint-src/src/tint/source.cc:31: undefined reference to `tint::text::utf8::Decode(unsigned char const*, unsigned long)'
/work/build-asan/_deps/tint-src/src/tint/source.cc:58: undefined reference to `tint::text::utf8::Decode(unsigned char const*, unsigned long)'
```

that happens even when I set `-DTINT_ENABLE_ASAN=ON`, and I'm not sure how much it's worth debugging further.